.github/workflows/eden.yml

```diff
@@ -0,0 +1,47 @@
+name: Eden
+on:
+  pull_request_review:
+    branches: [master]
+    types: [submitted]
+
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event.review.state == 'approved' }}
+    steps:
+      - name: setup
+        run: |
+          sudo apt update
+          sudo apt install -y qemu-utils qemu-system-x86
+      - name: get eden
+        uses: actions/checkout@v2
+        with:
+          repository: 'lf-edge/eden'
+      - name: build eden
+        run: |
+          make clean
+          make build
+          make build-tests
+      - name: get eve
+        uses: actions/checkout@v2
+        with:
+          path: 'eve'
+      - name: build eve
+        run: |
+          make -C eve HV=kvm eve
+      - name: run
+        run: |
+          TAG=$(find eve/dist/amd64 | grep kvm-amd64 | grep -o '0.*-kvm-amd64')
+          EVE_TAG=${TAG%-kvm-amd64}
```

Comment:
This part seems a bit brittle. Finding the tag by doing a search and then looking for the hard-coded `-kvm-amd64` is asking for trouble. It works for now (I used it in my tests too), but let's keep an eye on this for better ways.

At least it is put into an env var immediately, so we don't have to have it in multiple places.

Reply:
I created a simple PR #1534 to make it easy, so we can just do:

```
EVE_TAG=$(make -C eve version)
```

Reply:
Great point @deitch -- I merge #1534 and will update the PR here